src/index.html

```diff
@@ -1,13 +1,718 @@
+
+              <li class="feature__item">
+                Auto Music Playback from Last Song Stopped
+              </li>
+            </ul>
+          </article>
+
+          <article
+            class="features__feature feature features__feature--3"
+          >
+            <h3 class="feature__title">App features</h3>
+
+            <ul class="feature__list">
+              <li class="feature__item">
+                Customize Music Schedule
+              </li>
+
+              <li class="feature__item">
+                Wake Up with Favorite Songs
+              </li>
+
+              <li class="feature__item">
+                Home Detection Auto Wake Up
+              </li>
+
+              <li class="feature__item">
+                Color Wheel
+              </li>
+
+            </ul>
+          </article>
+
+          <div class="features__navigation">
+            <div class="features__navigation-arrows">
+              <div class="features__navigation-arrows-link-wrapper">
+                <a href="features__navigation-arrows-link--back">
+                  <svg
+                    width="48"
+                    height="13"
+                    viewBox="0 0 48 13"
+                    fill="none"
+                    xmlns="http://www.w3.org/2000/svg"
+                    class="features__navigation--pointer "
+                  >
+                    <path
+                      d="M48 4.19628e-06L0.329261 0L0.329261 0.999998L48 1V4.19628e-06Z"
+                      fill="#BDBDBD"
+                      class="features__navigation--pointer features__navigation-pointer--back"
+                    />
+                    <path
+                      d="M18.2022 11.8047L0.440509 0.000149576L0 1.19548L17.7616 13L18.2022 11.8047Z"
+                      fill="#BDBDBD"
+                      class="features__navigation--pointer features__navigation-pointer--back"
+                    >
+                  </svg>
+                </a>
+              </div>
+
+              <div class="features__navigation-arrows-link-wrapper">
+
+                <a href="features__navigation-arrows-link--forward">
+                  <svg
+                    width="48"
+                    height="13"
+                    viewBox="0 0 48 13"
+                    fill="none"
+                    xmlns="http://www.w3.org/2000/svg"
+                    class="features__navigation--pointer"
+                  >
+                    <path
+                      d="M0 13H47.6707V12H0V13Z"
+                      fill="#333333"
+                      class="features__navigation--pointer features__navigation-pointer--forward "
+                    />
+                    <path
+                      d="M29.7978 1.19533L47.5595 12.9999L48 11.8045L30.2384 0L29.7978 1.19533Z"
+                      fill="#333333"
+                      class="features__navigation--pointer features__navigation-pointer--forward"
+                    />
+                  </svg>
```

Comment:
Better just use img tag with a src to SVG file 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img